### ISSUE TITLE
Stylus support is broken

### ISSUE THREAD

**Issue by grandsong** [johnsoncodehk/volar#2006]
After I updated Volar to v1.0, completion for Stylus in Vue SFC became broken (Similar to #1027).
It seems that Volar 1.0 requires a new plugin (like "@volar/vue-language-plugin-pug") for Stylus.
Would you kindly make it?

**Comment by johnsoncodehk** [comment]
We have never support Stylus intellisense, Vetur support it a bit, I guess you disabled Vetur so loss Stylus intellisense.

Support Stylus by migrate Vetur Stylus supports code to https://github.com/johnsoncodehk/volar-plugins is possible, but I have no plan to do for now, and PR is welcome.

**Comment by grandsong** [comment]
In #29, you said "supported in 0.21.18."
And I did enjoyed functional Stylus support before (not using Vetur, but Volar of version < 1.0).
I thought you made it. No?

Anyway, I still hope you do the favor. Because,
1) You suggested a reasonable and universal way to support lang like Stylus. I think that would be ideal. 
2) I don't know if I can be capable of making a good one. If it is a considerble trouble for you, who seems knows well about how to do, it would be far more trouble for me...
3) Even if I managed to do that, and I published my plugin, it would be not easy for people to find it.
4) I am busy working. Meanwhile, the built-in emmet extension of VS Code partly helps. So I'd rather tolerate it for now.
